src/api/index.js

```diff
@@ -14,33 +16,26 @@ const getExampleData = () => {
     .then(response => response.data);
 };
 
-const getAuthHeader = authState => {
-  if (!authState.isAuthenticated) {
-    throw new Error('Not authenticated');
-  }
-  return { Authorization: `Bearer ${authState.idToken}` };
-};
-
-const getDSData = (url, authState) => {
```

Comment:
Nice job cleaning up the getDSData function. Even the smallest change can make the code look cleaner.

Reply:
Thank you